CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### Changed
+- added integration test for Datafusion `kamu sql` command
```

Comment:
Changelog is meant for external users of the tool or our libraries, so we usually don't mention internal improvements like adding tests in it.

Reply:
thank you, done